app/services.py

```
@@ -191,26 +305,43 @@ async def check_loan_eligibility(product_key: str) -> str:
+            quick_replies = await self._infer_next_actions(message, bot_message)
+            yield to_sse(
+                "done",
+                {
+                    "answer": bot_message,
+                    "quickReplies": quick_replies,
+                },
+            )
+
         except asyncio.CancelledError:
             raise
         except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)